recipes/gto/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "gto" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gto-{{ version }}.tar.gz
+  sha256: ce9501b6a7e283e331239fb9c957ff0170ff48df1c8402431309138c5d462556
+
+build:
+  entry_points:
+    - gto = gto.cli:app
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools-scm
+    - fastentrypoints >=0.12
+    - pip
+  run:
+    - python >=3.6
+    - gitpython
+    - typer >=0.4.1
+    - rich
+    - pydantic
+    - ruamel.yaml
+    - semver ==3.0.0-dev.3
```

your build fails because this `semver` version does not exist:

```
-   - nothing provides requested semver 3.0.0-dev.3
```

I also don't recommend pinning to the patch release. This will only cause problems when trying to resolve dependencies in larger environments.

```suggestion
    - semver ==3.*
```